src/lib/contents/docs/menu.ts

```diff
@@ -40,6 +40,7 @@ export const MENU: MenuEntry[] = [
     M("Collaboration & Sharing", "sharing-and-collaboration"),
     M("Teams & Projects", "teams-and-projects", [], "beta"),
     M("Team Billing", "team-billing", [], "beta"),
+    M("Usage-based Billing", "usage-based-billing", [], "beta"),
```

Comment:
In the [screenshot](https://github.com/gitpod-io/website/blob/23455e6a16caeb4fd96729d3afbdadd20156bb15/static/images/docs/billing/active-team-billing.png), we write it as `Usage-Based Billing` (basically in webApp). But, here it is written as `Usage-based Billing` 🤔 


```suggestion
    M("Usage-Based Billing", "usage-based-billing", [], "beta"),
```

Reply:
Thanks @Siddhant-K-code 

I'm going by this [guideline](https://titlecaseconverter.com/blog/capitalizing-hyphenated-words/) treating the word "based" in "Usage-based" as a minor word. The text in the screenshot should be fixed to conform to that as well :) 

> The first word in a hyphenated compound is capitalized, and the subsequent words are lowercased if they are minor words (in the respective style), and capitalized else